models/surveys.go

```diff
@@ -418,6 +436,26 @@ func (api *API) getSurveyID(surveyID string) error {
 	return api.GetSurveyIDStmt.QueryRow(surveyID).Scan(&id)
 }
 
+func isUUID(id string, idName string, w http.ResponseWriter) bool {
```

Comment:
Might be better to have this a function that attempts to read a UUID parameter rather than verify a string is a UUID, returning the UUID if ok and empty string + error if not.

e.g.
```
func getUUIDParam(name string, w http.ResponseWriter, r *http.Request) (uuid.UUID, error) {

}
```

Not sure if the function should do the output of the error to the client though